nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -320,6 +323,54 @@ TfOpNodes buildOpNodes(const GraphRepresentation &representation,
     layer_to_tf.insert({ln.get(), nodes.back().get()});
   }
 
+  int node_count = 0;
```

Comment:
I think what we have to do here is just identify which fully connected layer node needs to be reshaped. and do the reshaping in somewhere like saveTfLite